### PR TITLE
show detail order

### PR DESCRIPTION
**Pull request by ghost**
## Description
show detail order
## Reference Requirements
- Task ID: TI-1

## Migrations
YES
## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

```mysql> EXPLAIN select * from `orders` where `id` = '2' limit 1 \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: orders
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select * from `order_details` where `order_details`.`order_id` in ('2') \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: order_details
   partitions: NULL
         type: ref
possible_keys: order_details_order_id_foreign
          key: order_details_order_id_foreign
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.01 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select * from `users` where `users`.`id` in ('31') and `users`.`deleted_at` is null \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: users
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> EXPLAIN select * from `user_infos` where `user_infos`.`user_id` in ('31') and `user_infos`.`deleted_at` is null \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: user_infos
   partitions: NULL
         type: ref
possible_keys: user_infos_user_id_foreign
          key: user_infos_user_id_foreign
      key_len: 4
          ref: const
         rows: 1
     filtered: 10.00
        Extra: Using where
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified```

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

**Review by ghost — COMMENTED** [review]
please check my comments

File: resources/views/admin/order/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              <table class="table table-hover">
+                <tr>
+                  <th>{{__('order.admin.show.name_product')}}</th>
+                  <th>{{__('order.admin.show.quantity')}}</th>
+                  <th>{{__('order.admin.show.total')}}</th>
+                  <th>{{__('order.admin.show.image')}}</th>
+                  <th>{{__('order.admin.show.preview')}}</th>
+                  <th>{{__('order.admin.show.date')}}</th>
+                </tr>
+                  @foreach ($order->orderDetails as $item)
```
Comment (at-tuanphan) [inline resources/views/admin/order/show.blade.php]: $item -> orderDetail

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**